src/languages/en.js

```diff
@@ -284,7 +284,9 @@ export default {
         imageUploadFailed: 'Image upload failed',
         deleteWorkspaceError: 'Sorry, there was an unexpected problem deleting your workspace avatar.',
         sizeExceeded: ({maxUploadSizeInMB}) => `The selected image exceeds the maximum upload size of ${maxUploadSizeInMB}MB.`,
-        tooSmallResolution: ({minHeightInPx, minWidthInPx}) => `Please upload an image larger than ${minHeightInPx}x${minWidthInPx} pixels`,
+        resolutionConstraints: ({
+            minHeightInPx, minWidthInPx, maxHeightInPx, maxWidthInPx,
+        }) => `Please upload an image larger than ${minHeightInPx}x${minWidthInPx} and smaller than ${maxHeightInPx}x${maxWidthInPx} pixels`,
```

Comment:
I believe we used to ask for the exact copy to be used by adding [Waiting for copy](https://github.com/Expensify/App/labels/Waiting%20for%20copy) label in the issue. 

But the changes look good to me. @tgolen Your thoughts?

Reply:
Got an approval here: https://expensify.slack.com/archives/C01GTK53T8Q/p1672358985640889

Reply:
It's almost the same copy, so I think it's fine, but it's missing pixels:

```suggestion
        }) => `Please upload an image larger than ${minHeightInPx}x${minWidthInPx} pixels and smaller than ${maxHeightInPx}x${maxWidthInPx} pixels`,
```

Reply:
fixed